frontend/src/components/question-creation/question-elements/modals/multiple-choice/MultipleChoiceModal.tsx

```diff
@@ -61,7 +64,7 @@ const MultipleChoiceModal = ({
       setOptionCountError(true);
     } else if (correctOptionCount === 0) {
       setNoCorrectOptionError(true);
-    } else if (correctOptionCount > 1) {
+    } else if (!isMultiSelect && correctOptionCount > 1) {
```

Comment:
nit: instead of `!isMultiSelect` can we use `isMultipleChoice` here for more better readability?